### PR TITLE
Fix mkbootimg since 33.0.3 upgrade

### LINKED ISSUE #76: 33.0.3 breaks creating boot images with mkbootimg
With the latest android-tools 33.0.3, creating a boot image using mkbootimg is broken with this error:
```
Traceback (most recent call last):
File "/usr/bin/mkbootimg", line 31, in <module>
from gki.generate_gki_certificate import generate_gki_certificate
ModuleNotFoundError: No module named 'gki'
```
This issue does not happen in android-tools 31.0.3.

https://bugs.archlinux.org/task/76090 @anatol

### PR DESCRIPTION
**Pull request by JamiKettunen**
It now requires an import of a `gki/generate_gki_certificate.py`, so install the scripts in `/usr/share/android-tools/mkbootimg` and symlink back to `/usr/bin/mkbootimg` to avoid having to patch anything.

Suggestions for a better way to create the symlink are welcome, I'm not entirely happy with it, but it does work :)

Fixes #76.

### CODE REVIEW

**Comment by JamiKettunen** [comment]
Now that I thought about it `${CMAKE_INSTALL_FULL_DATADIR}/android-tools` (otherwise known as `/usr/share/android-tools`) would be a better place to put the scripts in than the rather arbitrary `/usr/lib/android-tools`

**Comment by JamiKettunen** [comment]
Would it be possible to also get #68 merged before creating a new `33.0.3p1` tag with this imho rather important fix included?

File: vendor/CMakeLists.avb.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+install(PROGRAMS avb/avbtool.py DESTINATION bin RENAME avbtool)
```
Comment (anatol) [inline vendor/CMakeLists.avb.txt:1]: It is better to split this PR into 2 separate commits.

1 - fix of mkbootimg
2 - enabling avb tool

Comment (JamiKettunen) [reply vendor/CMakeLists.avb.txt]: @anatol Done, see #79 for `avbtool`.

Comment (JamiKettunen) [reply vendor/CMakeLists.avb.txt]: Oh I just realized you meant splitting it to 2 commits in this PR, oh well... let me know if you want them both here and I can close the other.

Comment (anatol) [reply vendor/CMakeLists.avb.txt]: Either works for me.

The current condition looks good to me. There is no need to redo it.

@nmeum are you OK with this PR?

File: vendor/CMakeLists.mkbootimg.txt [hunk 2]
```diff
@@ -1,3 +1,9 @@
-install(PROGRAMS mkbootimg/mkbootimg.py DESTINATION bin RENAME mkbootimg)
+set(MKBOOTIMG_SCRIPTS_DIR "${CMAKE_INSTALL_FULL_DATADIR}/android-tools/mkbootimg")
+install(PROGRAMS mkbootimg/mkbootimg.py DESTINATION ${MKBOOTIMG_SCRIPTS_DIR})
+add_custom_target(mkbootimg_symlink ALL COMMAND ${CMAKE_COMMAND} -E create_symlink
```
Comment (anatol) [inline vendor/CMakeLists.mkbootimg.txt:3]: I see that this target is called even if I do not modify any sources

```
ninja: Entering directory `build'
[1/1] cd /home/anatol/sources/android-tools/build/vendor && /usr/bin/cmake -E create_symlink /usr/share/android-tools/mkbootimg/mkbootimg.py /home/anatol/sources/android-tools/build/vendor/mkbootimg
```

what is this custom target used for? Do we need it here?

Comment (JamiKettunen) [reply vendor/CMakeLists.mkbootimg.txt:3]: Nothing, this was one of the ways I found to create the final installed `/usr/bin/mkbootimg` symlink file itself (which the `install()` below places in `/usr/bin/`); I already acknowledged it wasn't so nice in the description of the PR.

Please do replace it with something else if you know a better way!

Comment (anatol) [reply vendor/CMakeLists.mkbootimg.txt:3]: I am not really an CMake expert cc @nmeum for more help

@nmeum Do you know how to get `install()` target to avoid re-run here in case of a no-op build?

Comment (nmeum) [reply vendor/CMakeLists.mkbootimg.txt:3]: `add_custom_target` is not intended to have any output and is thus always run. You could maybe try to work around that via [`add_custom_command`](https://cmake.org/cmake/help/latest/command/add_custom_command.html)? I haven't looked into using that to create symlinks as part of the installation though.
